src/exercicio16/Livro.java

```diff
@@ -0,0 +1,28 @@
+package exercicio16;
+
+public class Livro {
+    private String titulo;
+    private final String autor;

```

Comment:
Acho que atributos de Livro, Pessoa e Empréstimo não precisam ser final.
IntelliJ provavelmente reclama e sugere modificação das propriedades para final por não haver possibilidade prevista de alteração. Se incluir os setters das propriedades ele deixa de sugerir essa alteração, por exemplo.
public void setAutor(String autor) { this.autor = autor; }